Automattic/jetpack
# Whatsapp Block: Fix issue with button block height in FSE

Related to https://github.com/Automattic/jetpack/issues/19504

While testing the WhatsApp block in the new site editor in FSE, I noticed that the height was incorrect. The WhatsApp button block currently is sized expecting box-sizing to already be set to `border-box`, however in some FSE themes (e.g. TT1-blocks), we can't expect this CSS to be provided. This change adds it in to the block's view CSS.

| Before | After |
| --- | --- |
| ![image](https://user-images.githubusercontent.com/14988353/114656916-4797fa00-9d32-11eb-8234-6cbfe4e2fe54.png) | ![image](https://user-images.githubusercontent.com/14988353/114656922-4b2b8100-9d32-11eb-8b08-0e97691f749e.png) |

#### Changes proposed in this Pull Request:

* Add `box-sizing: border-box` to WhatsApp button block to ensure minimum height is applied correctly

#### Jetpack product discussion


https://github.com/Automattic/jetpack/issues/19504

#### Does this pull request change what data or activity we track or use?


No

#### Testing instructions:






* While running the Gutenberg trunk build, set your test site to use an FSE-compatible theme (TT1-blocks)
* Go to the site editor (`/wp-admin/admin.php?page=gutenberg-edit-site`)
* Insert a WhatsApp button block (Send a message) into the footer template part
* Check that the height of the block looks correct as in the After screenshot above
* Save the template part, and view the front end of the site
* Ensure the block is rendered with the correct height
* Double-check this change still looks okay in the post/page editor and on non-FSE themes
